extension/flagMapProducts.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function (context, input, cb) {
+  const triggerProps = [
+    {
+      label: 'Apply Map',
+      value: 'yes'
+    }
+  ]
+  for (let product of input.products) {
```

Comment:
You can do this much easier by using:
[`Array.filter`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/map) and [`Array.map`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)

Reply:
removed for loop and replaced it with the use of the map method of the Array class.